src/lib/viewers/doc/_docBase.scss

```diff
@@ -5,8 +5,18 @@ $thumbnail-border-radius: 4px;
 .bp {
     .bp-thumbnails-container {
         border-right: solid 1px $seesee;
+        bottom: 0;
         display: flex;
-        flex: 0 0 201px; // Accounts for the 1px border on the right so the remaining width is actually 180
+        flex: 0 0 auto;
+        left: -201px;
+        position: absolute;
+        top: 0;
+        transition: left 300ms cubic-bezier(.4, 0, .2, 1);
```

Comment:
I believe it's usually more efficient to animate `translate` vs `left/right/top/bottom`. Is that an option here?